src/Core/Domain/Address/Command/AddManufacturerAddressCommand.php

```diff
@@ -121,7 +127,8 @@ public function __construct(
         $stateId = null,
         $homePhone = null,
         $mobilePhone = null,
-        $other = null
+        $other = null,
+        $dni = null
```

Comment:
```suggestion
        ?string $dni = null
```

Reply:
It's 1.7.6.x, still compatible with 5.6. We can't type parameters.

Reply:
We can't in 176 (PHP5)